index.js

```diff
@@ -5,27 +5,28 @@ const Twitch = require('./lib/twitch')
 const CustomError = require('./lib/CustomError')
 const mongo = require('./lib/mongo')
 const fs = require('fs')
+const twitchApi = require('./lib/twitchApi')
 
 let twitchClient, dotaClient, mongoDb
 
-const self_channel = process.env.SELF_TWITCH_CHANNEL || '9kmmrbot'
-
 mongo.connect().then(c => {
 	mongoDb = c
-	twitchClient = new Twitch('9kmmrbot', 'oauth:' + process.env.TWITCH_AUTH)
+	twitchClient = new Twitch('9kmmrbot_staging', process.env.TWITCH_AUTH)
```

Comment:
Wait up, this (the username being my test bot's username) seems wrong. I'm not sure how, but the bot seems to log in and work fine regardless of username here. That being said, I should probably change it to look this name up from the API as well. Please hold off on merging this PR until I've done this.

Reply:
Fixed!